src/components/Settings/ProfileSetting.vue

```diff
@@ -71,6 +71,7 @@
   .p-float-label {
     margin-bottom: 10px;
   }
+
```

Comment:
probs no need for this line :D

Reply:
ah i see cool then yours win.

Reply:
but this file does not exist anymore, you need to merge develop into this branch.

Reply:
conflict resolved